content/src/fieldConfig/business-formation.json

```diff
@@ -205,6 +205,15 @@
         "label": "Original State of Formation",
         "error": "Select a State"
       },
+      "foreignGoodStandingFile": {
+        "label": "Foreign Certificate of Good Standing",
+        "helperText": "Select a PNG or PDF file. Max file size: 3MB.",
+        "errors": {
+          "fileSize": "${fileName} exceeds maximum size of ${maxFileSize} MB.",
+          "fileType": "${fileName} file format is not supported.",
+          "required": "A Certificate of Good Standing is required."
+        }
```

Comment:
Not a huge thing, but something for consideration: sometimes nesting in the CMS ui is helpful to organize things and close extraneous information. But the more nesting we do, the more grey bars and boxes that Netlify creates on the screen. 

So I tend to nest things when it is clear that you'd want to look at one thing while being able to collapse unrelated info. I would argue that a content team member looking at the Foreign Good Standing File would want to see all the fields, and there's not much value in being able to see label & helpertext but collapse the error fields. So at that point, the extra boxes and nesting in the CMS UI is more visually distracting than helpful. If you haven't looked at the CMS to see what I mean, I recommend it. Just a thoughtt.

Reply:
I haven't poked a ton around in there. I'll de-structure this.